perma_web/frontend/components/FolderCustomSelect.vue

```diff
@@ -50,6 +50,21 @@ const handleSelectToggle = () => {
     isSelectExpanded.value = !isSelectExpanded.value
 }
 
+const handleKeyboardSelectToggle = async (e) => {
+    const isButton = e.target.matches('button')
+
+    if (!isButton) {
+        return
+    }
+
+    if (isSelectExpanded.value) {
+        return handleArrowDown(e)
+    }
+
+    handleSelectToggle()
+    await nextTick()
```

Comment:
Note: `nextTick` is a built-in Vue function that is intended to make sure the app waits until all DOM updates occur. 

In practice, this means the application should always wait until the select menu is triggered before attempting to focus it.